### DISCUSSION TITLE
[Feature] Export Polygons for Visualization of the Labeling process as image

### DISCUSSION THREAD

**Discussion by andife** [wkentaro/labelme#1063] [Ideas / Feature requests]
**Is your feature request related to a problem? Please describe.**
In order to better verifty the label quality and the process of how the labels were created, it would be nice to see the background image with the polygons drawn in.

**Describe the solution you'd like**
Export functionality to png

**Describe alternatives you've considered**
Visualization of the masks directly (but with the overlay the boundaries are less clear)

**Additional context**

**Comment by wkentaro** [comment]
Why is a screenshot not enough?

**Comment by andife** [comment]
Thank you for the answer. Screenshots could be enough in themselves, but they would have to be done manually. If you just want to see a lot of labeled data, including how it was labeled, an automated export would be great.
